ui/widget.js

```diff
@@ -258,6 +259,7 @@ $.Widget.prototype = {
 		this.bindings = $();
 		this.hoverable = $();
 		this.focusable = $();
+		this.classObject = {};
```

Comment:
SOLD! :+1: 
when we adopt chassis ( and sass ) they can become `sassyElements` :-p